Samples/SqueezeNetObjectDetection/NET5/Labels.json

```diff
@@ -0,0 +1,1002 @@
+{
+  "0": "tench, Tinca tinca",
```

Comment:
same as https://github.com/microsoft/Windows-Machine-Learning/blob/master/Samples/SqueezeNetObjectDetection/Desktop/cpp/Labels.txt?

can we share?

Reply:
We can, currently it is copied into each project. But squeezenet should be refactored to have all common assets pulled out.